src/utils/Renderers.tsx

```diff
@@ -9,6 +9,7 @@ import * as Types from "./Types";
 /////////////////
 // MapChart.tsx
 /////////////////
+var rowCounter = 0;
```

Comment:
`var` is quite old to declare a variable.

Now you can use `let` or `const` to declare variables.
`let`: use this when you are expecting this variable will change
`const`: use this when you know this variable will not be changes